book/text.md

```diff
@@ -446,8 +448,8 @@ any accumulated text as a `Text` object. Otherwise, if you never saw
 an angle bracket, you'd return an empty list of tokens. But unfinished
 tags, like in `Hi!<hr`, are thrown out.[^errortag]
 
-[^errortag]: This may strike you as an odd decision: why not raise an error,
-    or finish up the tag for the author? I don't know, but dropping
+[^errortag]: This may strike you as an odd decision: why
```

Comment:
```suggestion
[^errortag]: This may strike you as an odd decision: why not
```

The footnote is trying to say, "If a document ends with `<foo`, one might imagine the browser should silently add a `>` character, so as not to lose data. But that's not what browsers do, they just drop the half-finished tag entirely."

Reply:
Right. My edit was to remove the part about "raise an error", because we already know that raising an error is a bad design pattern. Whether to silently add `>` OTOH is something that could have been decided on, so it makes sense to leave in the footnote.

Reply:
Right I'm just saying we have to say "why not finish the tag" since in fact browser's don't finish the tag. I think you just accidentally deleted a word.